docs/index.md

```diff
@@ -4,25 +4,25 @@ This package provides an interactive CLI and python client for [wandb.ai](https:
 
 ## Quickstart
 
-If you want to work with private buckets or push data to the cloud you must first [signup](https://app.wandb.ai/login)
+If you want to work with private runs or push data to the cloud you must first [signup](https://app.wandb.ai/login)
 
 ```console
 $ pip install wandb
 $ cd my_training_dir
 $ wandb init
-$ ./my_training.py arg1 | wandb my_bucket model.json weights.h5
+$ wandb my_script.py arg1
 ```
 
 To manually push files to the cloud run:
 
 ```console
-$ wandb push my-new-bucket somefile.proto
+$ wandb push latest somefile.proto
```

Comment:
I pushed https://github.com/wandb/core/commit/e6da6d4c774cf992cc0c996c5916b658ce23b386 which will find the latest run if the id is latest.

Reply:
"latest" is a possible auto-generated run id, so we'll either need to blacklist it from being generated or change run_id generation strategy. I'm going to make an issue about this.

Reply:
Good call 👍